### PR TITLE
contribution of tensorflow  erf frontend and  frontend tensorflow erf test function

### LINKED ISSUE #19786: erf

### PR DESCRIPTION
**Pull request by sharjeelnawaz8182**
# PR Description 
tenforflow frontend function erf  and  frontend tensorflow erf test function is added it this pull request as assignment pull request

## Related Issue

Close #19786

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: .idea/runConfigurations/_template__of_py_test.xml
Comment (YushaArif99) [inline .idea/runConfigurations/_template__of_py_test.xml:1]: can you kindly revert these changes 🙂

File: .idea/vcs.xml
Comment (YushaArif99) [inline .idea/vcs.xml:1]: same goes for here as well 🙂

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hello @sharjeelnawaz8182,

It appears that this PR has already been claimed. In fact, I was the assignee for the previous contributor who was working on this PR (https://github.com/unifyai/ivy/pull/19787).

Given that the PR is still under development, I would advise you to consider working on a different PR for now. I'm awaiting a response from the previous contributor of this PR. If there's no response in the coming days, I'll be closing that PR, at which point you would be free to take it up.

However, if your interest in this PR is related to the ML intern application, I would recommend choosing another PR since the majority of the work on this one has already been completed by the previous contributor.

Thank you! 🙂

**Comment by sharjeelnawaz8182** [comment]
@YushaArif99 this issue was open when i assigned it to my self .so please ,what are the points i should keep in mind before assigning issue to myself for ML intern application.

**Comment by Ishticode** [comment]
closing this as the requested changes have not been made and the test has strange indent leading to test collection errors. Please reopen if you have the fix. THank you very much.

**Pull request closed without merging**